### PR TITLE
update for use on Gnome 41 and 41.1

### PR DESCRIPTION
**Pull request by Yan-Solo**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "Organize windows in customizable snap zones like FancyZones on windows.",
   "version": 1.1,
   "shell-version": [
-    "40"
+    "40",
+    "41",
+    "41.1"
```
Comment (meronz) [inline metadata.json:9]: Just "41" works even on Gnome 41.1.

Comment (Yan-Solo) [reply metadata.json]: I was doubting that. First time doing this. Pushed another commit.

**Review by meronz — COMMENTED** [review]
You are committing unnecessary changes on the PR branch. You should reopen the pr only with the correct modifications...

**Comment by Yan-Solo** [comment]
Appearantly my recent commits have also been included. I don't know much about this since I'm no developer. Sorry for the hassle. It's working for me as I want it to work, so I guess I'll close this for now

**Comment by epizut** [comment]
Hi,

Even if the pull-request was containing other non-related commits and should be closed or modified this issue remains.
ie: gnome 41 support is still needed.

**Comment by meronz** [comment]
As a workaround, you can edit the installed manifest in `~/.local/share/gnome-shell/extensions/gSnap@micahosborne` and add "41" to the "shell-version" list.

**Comment by epizut** [comment]
I created a clean pr without any additional changes https://github.com/micahosborne/gSnap/pull/21

**Pull request closed without merging**